src/main/java/org/jenkinsci/plugins/githubautostatus/BuildStatusJobListener.java

```diff
@@ -83,10 +80,10 @@ public void onCompleted(Run<?, ?> build, TaskListener listener) {
             parameters.put(BuildNotifierConstants.BRANCH_NAME, statusAction.getBranchName());
 
             Result result = build.getResult();
-            statusAction.updateBuildStatusForJob(result == Result.SUCCESS
-                    ? BuildState.CompletedSuccess
-                    : BuildState.CompletedError,
-                    parameters);
+            if (result == null) {
+              result = Result.FAILURE;
```

Comment:
It might be worth logging an error here. Rereading the docs, it would be unexpected if we didn't have a result at this point